include/Server/src/Server.cpp

```diff
@@ -1,26 +1,19 @@
 #include "Server.h"
 
-#include <array>
-#include <ctime>
-#include <fstream>
-#include <iostream>
-#include <sodium.h>
-#include "../../ErrorHandler/include/ErrorHandler.h"
+constexpr std::size_t MAX_LENGTH = 1024;
```

Comment:
It is used in the code only 1 time, perhaps it would be better to make it a local variable